src/main/resources/init_db.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS books (
+    id INT NOT NULL AUTO_INCREMENT,
+    title VARCHAR(255),
+    price INT,
+    PRIMARY KEY (id)
+);
```

Comment:
It should be done like this:
```suggestion
    id BIGINT AUTO_INCREMENT PRIMARY KEY,
    title VARCHAR(255) NOT NULL,
    price DECIMAL(10, 2) NOT NULL
    );
```